packages/react-ui/src/components/HatchifyDataGrid/HatchifyDataGrid.tsx

```diff
@@ -54,7 +54,7 @@ function HatchifyCollection<
     [finalSchemas, schemaName],
   )
 
-  const collectionState = useCollectionState(
+  const DataGridState = useDataGridState(
```

Comment:
I'm not sure that this should be capitalized. That gives the impression that it is a component or type.

Reply:
Great catch, thank you